Source/player.cpp

```diff
@@ -817,8 +817,10 @@ void AddPlrMonstExper(int lvl, int exp, char pmask)
 		}
 	}
 
-	if (totplrs && (1 << myplr) & pmask) {
-		AddPlrExperience(myplr, lvl, exp / totplrs);
+	if (totplrs) {
+		exp = exp / totplrs;

```

Comment:
Does putting division into separate line changes the sequence of opcodes?

Reply:
Yes.

Long version, VC6 can optimize it away, vc5 cannot

Reply:
So, is the updated code also bin exact for v1.09b, in which case it should go into master?